### DISCUSSION TITLE
请问 [WinError 10061] 由于目标计算机积极拒绝，无法连接。是为什么呀QAQ

### DISCUSSION THREAD

**Discussion by butwedo** [Excaive/miraicle#13] [Q&A]
我尝试了关闭防火墙，修改了端口并且确保了端口没有被占用，同时也将网络代理设置改为了自动检测，但是还是会报错。。。

**Comment by Excaive** [comment]
不知道你有没有成功启动 `mirai-api-http`，可以在浏览器输入 `http://localhost:{port}/about` 看看有没有响应

**Reply by butwedo** [reply]
又重新装了一下`mirai-api-http`就好了，谢谢您！
